src/App.js

```diff
@@ -14,13 +14,12 @@ import { Layout, Menu } from 'antd';
 const { Header, Content, Footer } = Layout;
 
 class App extends Component {
-  componentDidMount() {
-    this.props.loadPosts();
-  }
+
+  // componentDidMount() {
```

Comment:
Feel free to delete this. Usually we don't leave commented-out code in repo since it will be present in an older commit.